library/core/src/mem/mod.rs

```diff
@@ -190,8 +190,8 @@ pub fn forget_unsized<T: ?Sized>(t: T) {
 ///
 /// Furthermore, `usize` and `isize` have the same size.
 ///
-/// The types `*const T`, `&T`, `Box<T>`, `Option<&T>`, and `Option<Box<T>>` all have
-/// the same size. If `T` is Sized, all of those types have the same size as `usize`.
+/// The types [`*const T`], `&T`, [`Box<T>`], [`Option<&T>`], and `Option<Box<T>>` all have
```

Comment:
the `*const T` link isn't reliable, sometimes it will go to `*mut T` instead. I put it behind a feature gate but apparently someone enabled it for core :/

Can you double check locally it goes to the right place? you can generate the docs with `x doc --open core`.

Reply:
It goes to [this](https://doc.rust-lang.org/std/primitive.pointer.html) which is both for const and mut pointers.

I have checked this before making the PR (making sure the links go to their intended places), but did it again now to make sure it's really going there.

Reply:
ah ok, you have a `[*const T]: pointer` reference link, I missed it. That kind of link is find, it's only the const/mut impls that are broken.